### ISSUE TITLE
HTMLElement.shadowRoot is deprecated.

### ISSUE THREAD

**Issue by ibqn** [mark-hahn/drag-drop-text#8]
The contents of `atom-text-editor` elements are no longer encapsulated
within a shadow DOM boundary. Please, stop using `shadowRoot` and access
the editor contents directly instead.
```
HTMLElement.shadowRoot (C:\Users\Evgen\AppData\Local\atom\app-1.19.0\resources\app\src\text-editor-element.js:13:10)
<unknown> (C:\Users\Evgen\.atom\packages\drag-drop-text\lib\drag-drop-text.coffee:29:45)
Workspace.observeTextEditors (<embedded>:26205:15)
DragDropText.activate (C:\Users\Evgen\.atom\packages\drag-drop-text\lib\drag-drop-text.coffee:26:43)
Package.activateNow (<embedded>:69370:25)
<unknown> (<embedded>:69342:38)
```

**Comment by utsavguptagit** [comment]
I got the issue as below

HTMLElement.shadowRoot - /snap/atom/247/usr/share/atom/resources/app.asar/src/text-editor-element.js:13:10
<unknown> - /home/classicsoft/.atom/packages/drag-drop-text/lib/drag-drop-text.coffee:29:45
Workspace.observeTextEditors - <embedded>:11:428320
DragDropText.activate - /home/user/.atom/packages/drag-drop-text/lib/drag-drop-text.coffee:26:43
Package.activateNow - <embedded>:11:3212513
measure - <embedded>:11:3211925

**Comment by mark-hahn** [comment]
Sorry, I abandoned atom a year ago and there is no support.

**Comment by keni8em** [comment]
Uploaded an update to the **_`drag-drop-text.coffee`_** file (#11). This bug can be marked as closed after merging my branch back with main and releasing the new version by @mark-hahn.

FYI: This was a bit trickier to fix. I removed **_`.shadowRoot`_** from line 18.

**Comment by keni8em** [comment]
Fixed in v1.0.0
